src/curve_fit.jl

```diff
@@ -196,5 +197,11 @@ function confidence_interval(fit::LsqFitResult, alpha=0.05; rtol::Real=NaN, atol
     confidence_intervals = collect(zip(coef(fit) - margin_of_errors, coef(fit) + margin_of_errors))
 end
 
+function f!_from_f(f, F::AbstractArray) #from NLSolversBase
```

Comment:
If you need this, why don't you just call it *from* NLSolversBase?

Reply:
I wasn't sure if it would work, I'm not very familiar with how Julia manages packages and modules

Reply:
`import NLSolversBase: f!_from_f` in `src/LsqFit.jl`